website/src/data/users.tsx

```diff
@@ -444,6 +444,14 @@ const Users: User[] = [
     source: 'https://github.com/CodeYourFuture/syllabus',
     tags: ['opensource', 'product'],
   },
+  {
+    title: 'Comp Labs',
+    description: 'Tech-based Blog, Enterprise-Grade Solutions and more...',
+    preview: require('./showcase/comp-labs.png'),
+    website: 'https://complabs.in/',
+    source: 'https://github.com/Comp-Labs/comp-labs-website',
+    tags: ['opensource', 'design', 'large', 'personal'],
```

Comment:
1. This site doesn't seem to have 200+ pages?
2. Is it personal?

Reply:
Yes... I seem to add personal and large tags by mistake and in a hurry. So these tags has to be removed